DGL_testing/README.md

```diff
@@ -0,0 +1,34 @@
+# Running DGL-PTM on GPU (snellius)
```

Comment:
can you please create a folder `docs` in the root and move this file there? can you also rename the file to something like `running_dgl_ptm_on_snellius_with_gpu.md`

Reply:
Done